hat/conf/play.conf

```diff
@@ -82,8 +82,18 @@ play {
       allowed = [".hubofallthings.net", ".hubat.net", ${?HAT_DOMAIN}]
     }
 
-    enabled = []
+    enabled = [
+        "com.github.stijndehaes.playprometheusfilters.filters.StatusCounterFilter",
+        "com.github.stijndehaes.playprometheusfilters.filters.StatusAndRouteLatencyFilter",
```

Comment:
there is a single filter that can do the job of thoese two

Reply:
Fixed.